### DISCUSSION TITLE
Ithere a way to use this software to convert m4b files to mp3 files?

### DISCUSSION THREAD

**Discussion by Murphino** [audiamus/AaxAudioConverter#181] [Q&A]
Been using this for my audible files for a while and recently started getting other audiobook files I would also like to "dumb down" for my non-smart mp3 player. Is there a way to use AAX to convert m4b files to mp3 files and preserve chapters as separate files just like it does for aax files? If preservation of chapters can't be done is there at least a way to convert?

Thank you!

**Comment by audiamus** [comment]
The straightforward way to convert .m4b to .mp3 is with FFmpeg directly, from a console window:

`ffmpeg.exe -hide_banner -i "C:\InPath\MyAudiobook.m4b" "C:\OutPath\MyAudiobook.mp3"`

I am not sure whether the chapter info will survive, the other meta data should. (Not many MP3 players understand chapter meta).

To use AAX Audio Converter you can apply a trick (the same trick that Book Lib Connect uses): Rename your .m4b file to .aax and then open it with AAX Audio Converter.

Or, if it is an Audible book, download it with [Book Lib Connect](https://github.com/audiamus/BookLibConnect).
